font/README.md

```diff
@@ -1,28 +1,13 @@
-This is the directory for Material Icons; the newer Material Symbols are at [../variablefont](https://github.com/google/material-design-icons/tree/master/variablefont).
+This directory contains the Material Symbols font files; the older Material Icons (non-variable) fonts are at [/material-icons](https://github.com/google/material-design-icons/tree/master/font/material-icons).
 
-Material Icons are the non-variable classic icon fonts, while the Material Symbols variable fonts offer weight, optical size, grade and fill variations (with grade and Fill being intended also for animated effects).
+The Material Symbols variable fonts offer weight, optical size, grade and fill variations (with grade and Fill being intended also for animated effects). Material Symbols are the non-variable classic icon fonts, while
```

Comment:
Paragraph ends in an incomplete sentence (resulting from copy-n-paste I think)

Reply:
Thanks! Fixed.
(Also fixed a code snippet that referenced the font incorrectly.)

Reply:
The issue is in the first <link> tag for Material Icons, which is missing a closing >; it should be <link.
```
href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">

```